rasa/nlu/featurizers/dense_featurizer/lm_featurizer.py

```diff
@@ -754,13 +796,19 @@ def train(
                     self._set_lm_features(batch_docs[index], ex, attribute)
                 batch_start_index += batch_size
 
-    def process(self, message: Message, **kwargs: Any) -> None:
-        """Process an incoming message by computing its tokens and dense features.
+    def process(self, messages: List[Message]) -> None:
+        """Processes a list of incoming message by computing their
+        tokens and dense features."""
+        for message in messages:
+            if message:
+                self._process_message(message)
```

Comment:
Specs say: **make sure to handle empty messages**
Is the `if` statement sufficient or do we mean something else?

Reply:
the loop handles empty lists as well so you're fine 👍🏻